.travis.yml

```diff
@@ -100,7 +100,8 @@ matrix:
           dist: trusty
           sudo: required
           compiler: gcc
-          node_js: "6"
```

Comment:
The first one should work though, it works on other projects. Than again, who understands the depths of Travis.